docs/shared-content-source/docs/modules/develop/pages/cloudflow-configuration.adoc

```diff
@@ -83,7 +83,10 @@ cloudflow {
 You need to specify at least one of the sections shown above.
 
 ==== Environment variables
-Environment / shell variables can be specified in the config as `${variable_name}`, `$variable_name`, or `${?variable_name}`. 
+Environment / shell variables can be specified in the config as `$\{variable_name}`, `$variable_name`, or `${?variable_name}`. 
+
+ifdef::review[REVIEWERS: I needed to escape the {} in the first variable_name above to prevent antora from complaining about an undefined variable.]
```

Comment:
perfectly aligned:
https://asciidoctor.org/docs/user-manual/#preventing-substitutions
but I think it should be escaped `${?variable_name}` as well, or am I wrong?

Reply:
It may be a bug in Antora, but I only needed to escape the first one. When i tried to do the remainder, the / appeared in the HTML output.

Reply:
Tried it again and still only need to escape the first.

Reply:
I trust you Ruth :-)